bin/test_daq

```diff
@@ -31,13 +31,13 @@ if [ -d faucet ]; then
 fi
 
 if [ -d inst ]; then
-    echo Directory inst/ exists, clean out before test run.
-    false
+    echo Directory inst/ exists, cleaning out before test run.
+    mv inst "inst.$(date)"
```

Comment:
why not the hard fail?  It was meant to a bad system test state.  When does this happen that you want to just move it?

Reply:
the config_base.sh writes out to inst/ before the test is run. I don't see the necessity to have one more environmental variable / other logic to indicate that config_base.sh should not write out to inst. I think it's more straight forward to backup users' old inst and local folders and create new ones to keep the test running.

Reply:
What does this have to do with the docker images?

Reply:
Currently there are 2 places where cmd/build is called. Once during setup time, once during aux test time. Since the correct docker images to be built are determined by the local system config, there needs to be a local and inst folders for that to work. This means, after running setup, there will be inst and local folder before cmd/run is called. Previously, you have specific logic to determine if the cmd/build is called under setup and selectively disable creating the inst and local folders so these checks from above will pass. I think that extra logic makes the workflow more complicated to understand.